README.md

```diff
@@ -2,9 +2,22 @@
 
 # Extension Workshop
 
-Launchpad for why and how to build Firefox extensions.
+Welcome to Extension Workshop, a launchpad for building Firefox extensions! 🚀
 
-## Getting Started
+## Updating Content
+
+If you would like to update content or other resources on Extension Workshop, please:
+
+    - Make a fork of this repository.
```

Comment:
You could link to these docs on how to do this: https://help.github.com/en/articles/fork-a-repo